OpenRA.Game/Traits/World/ActorMap.cs

```diff
@@ -88,24 +86,20 @@ public void Tick(ActorMap am)
 
 		class ProximityTrigger : IDisposable
 		{
-			public readonly int Id;
-			public WPos Position { get; private set; }
-			public WDist Range { get; private set; }
```

Comment:
You and @Mailaender have told me many times not to make something public unless it is being used by something.